js/search.js

```diff
@@ -36,7 +48,30 @@ function(ytData, playlist, template, $) {
     }
 
     var displaySearchResults = function(){
-        $("#search_res").empty()
+        $("#search_res").empty();
+        $("#search_res").append("<li><h6 class='ti'>Playlist</h6></li>");
+        if(searchPlaylist===null){
+            $("#search_res").append("<p>Nothing found</p>");
```

Comment:
When no search results in playlist, it would be better to not show the "Playlist" section at all.